ansible/playbooks/roles/common/tasks/main.yml

```
@@ -25,11 +25,11 @@
     - sysctl
 
 - name: Remove swap from /etc/fstab
-  mount:
-    backup: true
-    fstype: swap
-    path: swap
+  lineinfile:
+    path: /etc/fstab
+    regexp: ^((?!#).*\sswap\s.*\d\s+\d$)
```

```suggestion
    regexp: ^(?!#)\s*(?P<fs_spec>\S+)\s+(?P<fs_file>\S+)\s+(?P<fs_vfstype>swap)\b.*$
```
Longer version but with named groups since `fstab` has static format:
https://manpages.org/fstab/5